Library/Homebrew/brew.sh

```diff
@@ -410,6 +410,8 @@ Your Git executable: $(unset git && type -p $HOMEBREW_GIT)"
     fi
   fi
 
+  HOMEBREW_MINIMUM_GLIBC_VERSION="2.13"
```

Comment:
```suggestion
  HOMEBREW_LINUX_MINIMUM_GLIBC_VERSION="2.13"
```

or something to clarify this is only set on Linux?